cves/kernel/CVE-2020-27152.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  An issue was discovered in the Kernal-based virtual machines (KVM) module of the Linux Kernal
+  where an infinite loop would arise due to improper interaction between a resampler and edge 
+  triggering. This infinite loop could then be used by an attacker to consume excessive 
```

Comment:
would be a little better if there was more detail on some of the more technical terms- resampler/ edge triggering. maybe what an improper transaction could look like